internal/controller/messaging/command.go

```diff
@@ -25,15 +25,14 @@ import (
 )
 
 const (
-	CommandRequestTopic        = "CommandRequestTopic"
-	CommandResponseTopicPrefix = "CommandResponseTopicPrefix"
+	CommandRequestTopic = "CommandRequestTopic"
```

Comment:
do we have the constant from go.mod.contract?

Reply:
Good catch. Will update to use from core contracts.

Reply:
Actually, was already using one from core contracts, but missed removing this one. Removed now.